marimo/_save/loaders/loader.py

```diff
@@ -22,7 +25,10 @@
 )
 
 
-LoaderType = TypeVar("LoaderType", bound="Loader")
+class LoaderError(BaseException):
```

Comment:
Can you add a comment why this is subclasses `BaseException` and not `Exception`?